data_explorer/app/widgets.py

```diff
@@ -4,11 +4,12 @@
 import os
 from typing import Dict, List, Optional, Tuple, Union
 
+from fondant.filesystem import get_filesystem
```

Comment:
this import also needs a change I think ?

Reply:
no we still have src/filesystem.py so this should be fine